src/NuGetPackageVerifier/build/NuGetPackageVerifier.targets

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Target Name="VerifyPackages"
+          AfterTargets="Pack"
```

Comment:
Hmm, that's a usage I had not considered. I think maybe this PR is the wrong approach.  Instead of adding NuGetPackageVerifier as a target, it should be task. This task becomes available by taking a PackageDependency on NuGetPackageVerifier. But integration into the build is up to the consumer.

E.g. consumer usage
```xml
<!-- in Coherence-Signed.targets or KoreBuild.targets -->
<Project>
   <ItemGroup>
      <PackageDependency Include="NuGetPackageVerifier" Version="1.0.0" />
   </ItemGroup>

   <Target Name="VerifyPackages" DependsOnTargets="Pack">
      <NuGetPackageVerifier RuleFile="NPV.json" Directory="artifacts/build" />
   </Target>
</Project>
```

The problem, of course, is that you can't define a task without writing some C#. Last I heard, MSBuild for .NET Core doesn't support code tasks yet, so we'd have to compile/package a task. To keep it simple for now, though, that task can be super minimal and just shell execute NugetPackageVerifier.exe.

Thoughts?